<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsGui</class>
 <widget class="QWidget" name="ContactsGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget_contacts">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>340</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_age">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>340</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Age:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>240</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_save">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>410</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(30,144,255);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_lastName">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_firstName">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_uploadFile">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>470</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(220,20,60);
color:rgb(255,255,255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
</string>
       </property>
       <property name="text">
        <string>Upload File</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lastName">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>130</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>180</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_email">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_address">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>290</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_address">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>290</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_phone">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>180</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone Number:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_firstName">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_phoneErrorMessage">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_ageErrorMessage">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>370</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;.AppleSystemUIFont&quot;;</string>
       </property>
       <property name="text">
        <string>Enter your contact here </string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_firstName</tabstop>
  <tabstop>lineEdit_lastName</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_address</tabstop>
  <tabstop>lineEdit_age</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_uploadFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
